tests/h/views/admin/organizations_test.py

```diff
@@ -36,9 +37,9 @@ def test_it_returns_filtered_orgs(
         if query:
             pyramid_request.GET["q"] = query
 
-        ctx = index(None, pyramid_request)
+        response = index(None, pyramid_request)
```

Comment:
There are lots of very short variable names I don't think aid clarity.

Reply:
Yeah, `ctx` is a bit too much

Reply:
I actually legitimately don't know what it stands for here.

Reply:
It's usually short for `context`. In this case that might mean the "template context"?